packages/shared/lib/clients/orchestrator.ts

```diff
@@ -193,27 +214,6 @@ export class Orchestrator {
                 formattedError = new NangoError('action_failure', { error: errorToObject(err) });
             }
 
-            const content = `Action '${actionName}' failed`;
-            void logCtx.error(content, {
-                error: formattedError,
-                action: actionName,
-                connection: connection.connection_id,
-                integration: connection.provider_config_key
-            });
-            await logCtx.enrichOperation({ error: formattedError });
-
-            errorManager.report(err, {
-                source: ErrorSourceEnum.PLATFORM,
-                operation: LogActionEnum.SYNC_CLIENT,
-                environmentId: connection.environment_id,
-                metadata: {
-                    actionName,
-                    connectionDetails: JSON.stringify(connection),
-                    input
-                }
-            });
```

Comment:
I think this is redundant. we already set the span error tag below and attach meaningful tag at the top of the function